docs/usage_guide.rst

```diff
@@ -283,6 +283,25 @@ Here you can customize the precedence of querying that price oracles will have.
 DeFi oracles like Uniswap V2 and Uniswap V3 use only on chain information to get current prices, this means that it is a bit slower to query information from them but you depend only on the ethereum chain to query the prices.
 Also prices for some assets may differ from Coingecko or CryptoCompare depending on the conditions of the pools at the moment of making the query.
 
+
+NFT settings
+-----------------------
+
+.. image:: images/sc_nf_settings.png
+   :alt: Customizing the app's NFT settings
+   :align: center
+
+Include NFTs in graphs and total amounts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Configure whether to include NFTs in the calculation for the total net worth and the graphs displayed in the dashboard and statistic section.
+
+NFT images rendering setting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For privacy concerns, it is possible to allow all or only a certain list of domains for images rendered.
```

Comment:
Can you also add a link to the article describing the problem this setting aims to tackle? Here: https://medium.com/@alxlpsc/critical-privacy-vulnerability-getting-exposed-by-metamask-693c63c2ce94

Reply:
Ok